src/distilabel/utils/dataset.py

```diff
@@ -51,3 +60,215 @@ def load_task_from_disk(path: Path) -> "Task":
     with open(task_path, "rb") as f:
         task = pickle.loads(f.read())
     return task
+
+
+def _binarize_dataset(
+    dataset: "CustomDataset",
+    seed: int = None,
+    strategy: BinarizationStrategies = "random",
+    keep_ties: bool = True,
+    **kwargs: Any,
+) -> "CustomDataset":
+    """Binarizes a distilabel dataset.
```

Comment:
again, some people might be like: "what is binarizing?"

Reply:
Could you both @davidberenstein1957 and @dvsrepo take a look at the new section of the docs for the dataset preparation/binarization? This commit https://github.com/argilla-io/distilabel/pull/264/commits/e4602d4e2c0a68737915cfe6a743ead343091d4e contains the updates.